lib/Chat/CommentsManager.php

```diff
@@ -92,4 +92,30 @@ public function retrieveReactionsByActor(string $actorType, string $actorId, arr
 
 		return $reactions;
 	}
+
+	/**
+	 * @param integer $roomId
+	 * @param \DateTime $min
+	 * @param \DateTime $max
+	 * @return int[]
+	 */
+	public function getMessageIdsByRoomIdInDateInterval(int $roomId, \DateTime $min, \DateTime $max): array {
+		$query = $this->dbConn->getQueryBuilder();
+		$query->select('id')
+			->from('comments')
+			->where(
+				$query->expr()->andX(
+					$query->expr()->eq('object_id', $query->createNamedParameter($roomId, IQueryBuilder::PARAM_INT)),
+					$query->expr()->eq('object_type', $query->createNamedParameter('chat')),
+					$query->expr()->gte('creation_timestamp', $query->createNamedParameter($min, IQueryBuilder::PARAM_DATE)),
+					$query->expr()->lte('creation_timestamp', $query->createNamedParameter($max, IQueryBuilder::PARAM_DATE))
+				)
```

Comment:
Should also limit to some messages only?
Would be nice to keep that a user was added?

Then again, if we turn around the logic and add the ttl time when the messages is being created we can control it for each message and just not set a TTL for those.